frontend/00-browsertrix-nginx-init.sh

```diff
@@ -12,6 +12,18 @@ else
   sed -i "s/\$LOCAL_BUCKET/$LOCAL_BUCKET/g" /etc/nginx/includes/minio.conf
 fi
 
+# Plausible analytics
+if [ -z "$ANALYTICS" ]; then
+  echo "analytics enabled, injecting script"
+  [[ "$ANALYTICS" = "true" ]] && SRC="https://p.webrecorder.net" || SRC="$ANALYTICS"
+
+  # Manually minified/compressed version of scripts/inject-analytics.js
+  echo "let a=document.createElement('script');a.src='$SRC/js/script.file-downloads.hash.pageview-props.tagged-events.js',a.defer=!0,a.dataset.domain='browsertrix.com',document.head.appendChild(a);let b=document.createElement('script');b.textContent='window.plausible=window.plausible||function(){(window.plausible.q=window.plausible.q||[]).push(arguments)}',document.head.appendChild(b);" > /usr/share/nginx/html/_plausible.js
```

Comment:
Hm, would probably be better to just copy (and minify) inject-analytics.js here to avoid this getting out of sync?

Reply:
Yeah maybe, but idk, that becomes yet another build step & this file becomes a generated artifact & we need templating to generate it. I'm not sure this is going to change often enough to bother?

Reply:
I'll make a note that it's not synced in `inject-analytics.js` for clarity, but I don't think it's worth the extra effort & complexity

Reply:
If we don't minify (do we need to?) could just do something like `COPY ./scripts/inject-analytics.js` in Dockerfile and then `cp ./inject-analytics.js /usr/share/nginx/html/_plausible.js` ?

Reply:
Actually, may be a better option is just to put that analytics in the env var, INJECT_ANALYTICS, and then do:
```echo $INJECT_ANALYTICS > /usr/share/nginx/html/_plausible.js``` and we can just put that into our helm chart values, and someone else can use their own, if they'd like.